.github/workflows/rebake.yml

```
@@ -0,0 +1,118 @@
+      actions: "write"
+      contents: "write"
+      packages: "read"
+      pull-requests: "write"
+    container: "ghcr.io/nautobot/cookiecutter-nautobot-app-drift-manager/prod:${{ github.event.inputs.drift-manager-tag }}"
+    env:
+      GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+    steps:
+      - name: "Configure Rebake Arguments"
+        id: "config"
+        shell: "bash"
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:36:22: Double quote to prevent globbing and word splitting

(shellcheck)